pom.xml

```diff
@@ -1025,51 +1026,374 @@
                                         <binaryCompatible>true</binaryCompatible>
                                         <sourceCompatible>true</sourceCompatible>
                                     </overrideCompatibilityChangeParameter>
                                 </overrideCompatibilityChangeParameters>
                                 <includes>
-                                    <include>org.wildfly.security</include>
-                                    <include>org.wildfly.security.asn1</include>
-                                    <include>org.wildfly.security.auth</include>
-                                    <include>org.wildfly.security.auth.callback</include>
-                                    <include>org.wildfly.security.auth.client</include>
-                                    <include>org.wildfly.security.auth.jaspi</include>
-                                    <include>org.wildfly.security.auth.permission</include>
-                                    <include>org.wildfly.security.auth.principal</include>
-                                    <include>org.wildfly.security.auth.server</include>
-                                    <include>org.wildfly.security.auth.server.event</include>
-                                    <include>org.wildfly.security.auth.util</include>
-                                    <include>org.wildfly.security.authz</include>
-                                    <include>org.wildfly.security.credential</include>
-                                    <include>org.wildfly.security.credential.source</include>
-                                    <include>org.wildfly.security.credential.store</include>
-                                    <include>org.wildfly.security.evidence</include>
-                                    <include>org.wildfly.security.http</include>
-                                    <include>org.wildfly.security.key</include>
-                                    <include>org.wildfly.security.manager</include>
-                                    <include>org.wildfly.security.manager.action</include>
-                                    <include>org.wildfly.security.mechanism</include>
-                                    <include>org.wildfly.security.password</include>
-                                    <include>org.wildfly.security.password.interfaces</include>
-                                    <include>org.wildfly.security.password.spec</include>
-                                    <include>org.wildfly.security.permission</include>
-                                    <include>org.wildfly.security.sasl.util</include>
-                                    <include>org.wildfly.security.ssl</include>
-                                    <include>org.wildfly.security.x500</include>
-                                    <include>org.wildfly.security.x500.cert</include>
+                                    <include>org.wildfly.security.*</include>
```

Comment:
Is there a reason you needed to add the '*' wildcard?  Looking at the report I think these may be bringing in children to the test that should not be coming in.

Reply:
You're right, I will remove the wildcards.